frontend/components/FieldMedia/FieldMediaEdit.jsx

```diff
@@ -137,103 +137,131 @@ export default class FieldImageEdit extends Component {
     const singleFile = schema.settings && schema.settings.limit === 1
     const values = (value && !Array.isArray(value)) ? [value] : value
 
+
     return (
-      <Label label={displayName}>
+      <Label label={displayName} className={styles.label}>
         {values &&
-          <div class={styles['value-container']}>
-            <div class={styles.thumbnails}>
-              {values.map(value => (
-                <img
-                  class={styles.thumbnail}
-                  src={this.getImageSrc(value)}
+          values.map((value, idx) => {
+
+            let file = {}
```

Comment:
Just curious, could we not declare all three properties as part of the variable initialisation?